molecule/default/verify.yml

```diff
@@ -30,6 +30,7 @@
     - name: Check Runner
       assert:
         that:
+          - registered_runners.json.runners|json_query('labels[*].name')|flatten|sort == ['Linux', 'self-hosted', 'x86']
```

Comment:
I wanted to test if only default labels are used by default (no value for `runner_labels` provided), but I don't know how run this test :(